### PR TITLE
Update Debian-Chroot to Jessie

### PR DESCRIPTION
**Pull request by rmsr**
- Update version to 8.0-1
- Update base distro to Jessie
- Add commented-out testing entries in apt.sources, and comment out unstable
  and experimental. Apt-pinning is never 100% and should not be on by default.
- Clean preferences of references to obsolete third-party repos

### CODE REVIEW

File: spk/debian-chroot/Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 SPK_NAME = debian-chroot
-SPK_VERS = 7.4
-SPK_REV = 5
+SPK_VERS = 8.1
+SPK_REV = 1
```
Comment (Dr-Bean) [inline spk/debian-chroot/Makefile]: `SPK_REV` only goes up.

File: spk/debian-chroot/src/sources.list [hunk 2]
```diff
@@ -1,31 +1,33 @@
...
+# (unless you change the ordering). See /etc/apt/preferences for more
+# information on how pinning is set up.
 
 ################################################
 ## jessie
-deb http://ftp.fr.debian.org/debian/ jessie main contrib non-free
-deb-src http://ftp.fr.debian.org/debian/ jessie main contrib non-free
-
-## jessie security
+deb http://ftp.debian.org/debian/ jessie main contrib non-free
```
Comment (Dr-Bean) [inline spk/debian-chroot/src/sources.list]: It's bad netiquette to use the central servers. Use mirrors where possible, which is why `fr` was used.
Advanced users should probably change the defaults at some point anyway (unless they live in France...)

**Comment by Dr-Bean** [comment]
Added a couple comments. I don't have a particular opinion on the apt-pinning changes, I've never run into issues with it myself. I guess I'm fine with it either way. @Diaoul?

Anyway, thanks for the contribution, much appreciated :)

**Comment by Diaoul** [comment]
LGTM

File: spk/debian-chroot/src/sources.list [hunk 3]
```diff
@@ -1,31 +1,33 @@
...
 ## jessie
 deb http://ftp.fr.debian.org/debian/ jessie main contrib non-free
 deb-src http://ftp.fr.debian.org/debian/ jessie main contrib non-free
-
-## jessie security
 deb http://security.debian.org/ jessie/updates main contrib non-free
 deb-src http://security.debian.org/ jessie/updates main contrib non-free
 
+# jessie-updates - optional, but default
+deb http://ftp.fr.debian.org/ jessie-updates main contrib non-free
```
Comment (Dr-Bean) [inline spk/debian-chroot/src/sources.list]: This line and L14 are missing the `debian/` part if I'm not mistaken: `deb http://ftp.fr.debian.org/debian/ jessie-updates main contrib non-free`

**Comment by Dr-Bean** [comment]
While testing this, it seems `aptitude` is no longer available by default, meaning the help file is incorrect. Can't find a source on that, but it's simply not installed afaict. 
We should be able to replace the current commands in the help file with `apt-get <command>`.
@Diaoul On that note, mind if we put the current help file in the Github wiki, and point to that via `HELPURL`? Probably better than having a non-working link in the package.

File: spk/debian-chroot/src/sources.list (hunk 3 above)
Comment (rmsr) [inline spk/debian-chroot/src/sources.list]: Multiple lines missing that, fix pushed.

**Comment by rmsr** [comment]
aptitude is used to maintain and upgrade the debian chroot, so that's another thing that needs fixing.

**Comment by Dr-Bean** [comment]
Thanks :) Made a minor correction in the helpfile with https://github.com/SynoCommunity/spksrc/commit/3282e03de643b10e701c2996136b26d8c3e17722. 
Also, switched over to a wiki page: https://github.com/SynoCommunity/spksrc/wiki/Debian-Chroot. Can be moved over to https://synocommunity.com whenever, but for now, this produces a working link in the package.

**Pull request merged**